.github/scripts/build_npm_applications.sh

```diff
@@ -0,0 +1,46 @@
+PRE_OR_POST_RELEASE=$1
+MODULE_TYPE=$2
+
+
+if [ "$PRE_OR_POST_RELEASE" = "PRE" ]; then 
+    if [ "$MODULE_TYPE" = "NPM-ES" ]; then  
+        cd smoke/smoke-test-application-NPM-ES
+        npm uninstall aws-rum-web
+        npm run clean 
+        npm install 
+        npm install /home/runner/work/aws-rum-web/aws-rum-web #Install locally 
+        npm run build
+    elif [ "$MODULE_TYPE" = "NPM-CJS" ]; then  
+        cd smoke/smoke-test-application-NPM-CJS
+        npm uninstall aws-rum-web
+        npm run clean 
+        npm install 
+        npm install $(npm pack /home/runner/work/aws-rum-web/aws-rum-web | tail -1) #Install locally 
+        npm run build
+    else
+        echo "Not a valid module type"
+    fi
+elif [ "$PRE_OR_POST_RELEASE" = "POST" ]; then
+
+    if [ "$MODULE_TYPE" = "NPM-ES" ]; then  
+        cd smoke/smoke-test-application-NPM-ES
+        npm uninstall aws-rum-web
+        npm run clean 
+        npm install 
+        npm install aws-rum-web #Install latest released direct from NPM 
```

Comment:
nit: We already installed aws-rum-web in the previous `npm install` step. Here and elsewhere.

Reply:
OK I see what you're doing here. Since we've specified the actual version of the web client (i.e., `1.13.6` instead of `*`), we need to run a separate `npm install aws-rum-web`.

Reply:
Yes `npm install ` should get the latest released rum web client version. This just double confirms the behavior and if the new version builds correctly, before ending the release flow. 
I also npm uninstall to completely ensure the installation is that of the latest version and there is nothing left from the last version